extensions/libarchive/MergeContent.h

```diff
@@ -43,6 +43,8 @@ namespace processors {
 #define MERGE_FORMAT_AVRO_VALUE "Avro"
 #define DELIMITER_STRATEGY_FILENAME "Filename"
 #define DELIMITER_STRATEGY_TEXT "Text"
+#define ATTRIBUTE_STRATEGY_KEEP_COMMON "Keep Only Common Attributes"
+#define ATTRIBUTE_STRATEGY_KEEP_ALL_UNIQUE "Keep All Unique Attributes"
```

Comment:
I am not too fond of having these as defines. Maybe they belong under `MergeContent` or `BinFiles` as `static constexpr const char*`-s.